src/loaders/ImageLoader.js

```diff
@@ -26,7 +30,7 @@ Object.assign( THREE.ImageLoader.prototype, {
 
 		} else {
 
-			var loader = new THREE.XHRLoader( this.manager );
```

Comment:
The ImageLoader will call the methods itemStart and itemEnd of its loadingManager and the XHRLoader/FileLoader will call the same methods of a default loading Manager. I think it would be nice to let the FileLoader use the loading manager and this way we can store all active requests and abort them all.